app/code/Magento/Contact/Api/MailInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Contact module base controller
+ */
+namespace Magento\Contact\Api;
+
+interface MailInterface
+{
+    /**
+     * Send email from contact form
+     *
+     * @param string $replyTo Reply-to email address
+     * @param array $variables Email template variables
```

Comment:
but email template variables can be anything, from strings and integers to objects. How would you deal with it? `mixed[]` is as unspecific as `array`, so the only thing I can come up with is to hide the unspecifiy behind a `DataObject`.